architecture/tube.go

```diff
@@ -213,3 +214,33 @@ func (tube *Tube) Bury(command *Command) {
 		command.Err = errors.New(NOT_FOUND)
 	}
 }
+
+func (tube *Tube) Kick(command *Command) {
+	amount := 0
+	bound, err := strconv.Atoi(command.Params["bound"])
+	if err != nil {
+		// handle non-integer number of jobs to kick
```

Comment:
For now we have an array in the config of each command specifying command name, I think good idea would be to add an additional array under it that would specify expected type. Then we can implement some validation. Let me know what are your thoughts guys @vimukthi-git and @stephenwithav .

https://github.com/vimukthi-git/beanstalkg/blob/31697b380513b153cd8a57fb66507af423f72bcd/architecture/protocol_config.go#L68